packages/ton-client/src/utils/gesture.ts

```diff
@@ -0,0 +1,44 @@
+import { vec2 } from 'gl-matrix';
```

Comment:
this file should not exist anymore ;)

Reply:
Oh yea, I saw it marked as "they removed it" in the merge conflict. 
Will remove it.